probes/sastToolCodeQLInstalled/def.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: sastToolCodeQLInstalled
+short: Check that the project uses the CodeQL github actions
+motivation: >
+  SAST is testing run on source code before the application is run. Using SAST tools can prevent known classes of bugs from being inadvertently introduced in the codebase.
+implementation: >
+  The implementation checks whether the project invokes the github/codeql-action/analyze workflow.
```

Comment:
nit: `github/codeql-action/analyze` is a GitHub action, not a workflow

Reply:
Corrected in https://github.com/ossf/scorecard/pull/3571/commits/2f9f8e017285a515e34dbd15bb72f730a91d8a7c